test/clonevolume_tester.py

```diff
@@ -44,6 +45,7 @@ def setup_mock_objects(self):
 
         mock_3parclient = self.mock_objects['mock_3parclient']
         mock_3parclient.copyVolume.return_value = {'taskid': data.TASK_ID}
+        mock_3parclient.isOnlinePhysicalCopy.return_value = False
```

Comment:
It will be nice to have one unit test to return 'True' and see if the create snapshot returns the "Error string"